src/DataFixtures/ORM/LoadMissionTypeData.php

```diff
@@ -22,6 +23,12 @@ class LoadMissionTypeData implements FixtureInterface
 
     public function load(ObjectManager $manager)
     {
+        if (!$manager instanceof  EntityManagerInterface) {
```

Comment:
you may use a trait here, with a function which takes 2 args : $manger, $tableName

Reply:
`$manger` & `$tableName` are great var names, same lexical field and all... :trollface:

Reply:
I do agree

Reply:
with @salahm lol